src/Api/Endpoints/Login.php

```diff
@@ -34,7 +34,7 @@ class Login extends Endpoint
      *
      * @return boolean true if access is permitted
      */
-    function _hasAccess(\User $user) : bool
+    private function hasAccess(\User $user) : bool
```

Comment:
shouldn't this be public?

Reply:
Probably? I was just going based off of the underscore.

Reply:
PEAR standards used `_` for both protected and private.. based on the underscore, it should be protected (if that still works.. but I don't see why you'd want to prevent another class from checking user access)

Reply:
Alright I'll change it to protected